packages/runner/cypress/integration/sessions.ui.spec.js

```diff
@@ -19,7 +19,7 @@ const validateCreateNewSessionGroup = () => {
   .should('have.class', 'command-is-open')
   .contains('runSetup')
 
-  cy.contains('Create New Session')
+  return cy.contains('Create New Session')
```

Comment:
```suggestion
  cy.contains('Create New Session')
```

It's never necessary to return a Cypress chain.